connexion/api.py

```diff
@@ -161,11 +161,16 @@ def add_paths(self, paths=None):
                     if self.debug:
                         logger.exception(error_msg)
                     else:
-                        import sys
                         logger.error(error_msg)
+                        import sys
                         et, ei, tb = sys.exc_info()
-                        raise ei.with_traceback(tb)
-
+                        if sys.version_info[0] == 3:
+                            raise ei.with_traceback(tb)
+                        else:
+                            raise et, "Fatal error found while addding '{}': {}".format(
+                                        endpoint.get('operationId'),
+                                        str(ei)), tb
+                                        
```

Comment:
According to the tests this line (173) has unnecessary spaces, that's what's making flake8 fail.